packages/api/src/activities/utils/remove-mentions-text.ts

```diff
@@ -0,0 +1,51 @@
+import { MessageSendActivity, MessageUpdateActivity } from '../message';
+import { TypingActivity } from '../typing';
+
+/**
+ * any activity type that has a `text` property
+ */
+type TextActivity = MessageSendActivity | MessageUpdateActivity | TypingActivity;
+
+export interface RemoveMentionsTextOptions {
+  /**
+   * the account to remove mentions for
+   */
+  accountId?: string;
```

Comment:
Hm, consider using `entity`. Account isn't verbiage we normally use here right? Looks like activity uses `entity`

Reply:
in BF they use the mentioned accounts id, https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Schema/ActivityExtensions.cs#L64, and entities don't typically have a unique identifier.